### PR TITLE
Improve bot commenting

### LINKED ISSUE #1446: Comment all bot infrastructure interfaces
Some bots conform to commenting their parameters (liquidator and dispute) but some done (create price feed). All infra should be consistently commented.

### PR DESCRIPTION
**Pull request by chrismaree**
This PR standardizes the commenting across all bots, modules, and clients in our ecosystem. 

close #1446

### CODE REVIEW

File: disputer/disputer.js [hunk 1]
```diff
@@ -169,9 +165,6 @@ class Disputer {
         disputeResult: logResult
       });
     }
-
-    // Update the EMP Client since we disputed liquidations.
-    await this.empClient.update();
```
Comment (chrismaree) [inline disputer/disputer.js:174]: these update calls I removed as they are redundant in this bot. update is always called at the start of the operation so no need to update it after polling action is finished.

File: disputer/index.js [hunk 2]
```diff
@@ -20,12 +20,13 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
...
- * @param {Number} pollingDelay number of seconds to delay between polls.
- * @param {Object} priceFeedConfig configuration to construct the price feed object.
+ * @param {bool} shouldPoll Whether to poll continuously or run a single iteration (for testing).
+ * @param {Number} pollingDelay Number of seconds to delay between polls.
+ * @param {Object} priceFeedConfig Configuration to construct the price feed object.
+ * @param {Object} [disputerConfig] Configuration to construct the disputer.
  * @return None or throws an Error.
  */
-async function run(address, shouldPoll, pollingDelay, priceFeedConfig) {
+async function run(address, shouldPoll, pollingDelay, priceFeedConfig, disputerConfig) {
```
Comment (chrismaree) [inline disputer/index.js:29]: this optional config was missing

Comment (nicholaspai) [reply disputer/index.js:29]: wow good catch

File: liquidator/index.js [hunk 3]
```diff
@@ -18,17 +18,14 @@ const { Networker } = require("../financial-templates-lib/price-feed/Networker")
 const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
 const ExpandedERC20 = artifacts.require("ExpandedERC20");
 
-// TODO: Figure out a good way to run this script, maybe with a wrapper shell script.
-// Currently, you can run it with `truffle exec ../liquidator/index.js --address=<address> --price=<price>` *from the core
-// directory*.
-
 /**
- * @notice Continuously attempts to liquidate positions in the EMP contract.å
```
Comment (nicholaspai) [inline liquidator/index.js:26]: Haha nice catch again

**Pull request merged**